### DISCUSSION TITLE
releases and packages badges

### DISCUSSION THREAD

**Discussion by sombriks** [community/community#82188] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

there is a dedicated button inside actions where i can copy a very nice badge stating last workflow run status
it would be nice to have something similar for releases and packages versions.

right now i can grab version badges for 3rd party registries but not for official github ones. at least not easily.

**Comment by Imadnajam** [comment]
I can easily obtain version badges for 3rd party registries, but obtaining badges for official GitHub releases and packages is not as straightforward. Having a dedicated button or a similar feature to generate badges for releases and package versions within GitHub would greatly enhance user experience and streamline the process.
